Sources/Sentry/SentryNetworkTracker.m

```diff
@@ -456,8 +456,9 @@ - (void)captureFailedRequests:(NSURLSessionTask *)sessionTask
     context[@"response"] = response;
 
     if (self.isGraphQLOperationTrackingEnabled) {
-        context[@"graphql_operation_name"] =
-            [URLSessionTaskHelper getGraphQLOperationNameFrom:sessionTask];
+        NSMutableDictionary<NSString *, id> *graphql = [[NSMutableDictionary alloc] init];
+        context[@"graphql"] = graphql;
+        graphql[@"operation_name"] = [URLSessionTaskHelper getGraphQLOperationNameFrom:sessionTask];
```

Comment:
Not sure if there's anything special needed to get prettier names in the Sentry dashboard, or if that's necessary. Here's how this looks in a test event I generated with our app:

![image](https://github.com/user-attachments/assets/47c131f2-87fb-4b3d-9414-0bc34a8ae5b0)

`GraphQL` and `Operation Name` would look more aligned with the `Operating System` context, but my hunch is these may be getting special treatment server-side since I don't see these strings in the `sentry-cocoa` project anywhere.